### DISCUSSION TITLE
Can't retrieve more than one cookie when testing

### DISCUSSION THREAD

**Discussion by amb-lucas** [Kong/insomnia#4922] [Ideas] [answered]
### Context
I'm trying to test a request which expects multiple cookies in the response.

### Issue
I currently fetch the cookies via the response headers (response.headers['set-cookie']), but because the headers are parsed as an object, the headers['set-cookie'] is overwritten.
In other words, when a response comes back with two ['set-cookie'] headers, only the last cookie can be fetched via the request.headers object.

I tried to look for a workaround as to how to directly read the cookies from the testing script, but couldn't find anything :/

### Steps to reproduce
1. Make a request which has a response with multiple cookies being set
2. Try to access them via testing scripts (response.headers['set-cookie'] is the way I know how)
3. The first cookie set shouldn't be accessible/visible

**Comment by filfreire** [comment, marked as answer]
Duplicate of https://github.com/Kong/insomnia/issues/4923
